packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -266,15 +258,15 @@ export class SpecBuilder {
 
   userUnlistens(query: Query): this {
     this.nextStep();
-    if (!objUtils.contains(this.queryMapping, query.canonicalId())) {
+    if (!objUtils.contains(this.queryMapping, query.toTarget().canonicalId())) {
```

Comment:
Nit: This could be simplified if you asserted that targetId is not undefined after line 264. I am mostly just pointing this out to see if we can remove calls to `toTarget()`.

Reply:
No longer calling canonicalId.